### DISCUSSION TITLE
Error facing in creating service plan After upgrade from 3.18. To 3.19

### DISCUSSION THREAD

**Discussion by nspinternet** [hotspotbilling/phpnuxbill#138] [Q&A] [answered]
Please Remember, this project is free and open source, and @ibnux don't get any money from this project, and if you post something not a bug, just you dont understand how to install it, you will get blocked from this Repository.
Post it in Discussion if you don't understand. Except you pay for $50 for support

**Describe the bug**
A clear and concise description of what the bug is. Error connecting to router is not a bug, is your router port is not accessable, ask community for help, go to discussion or telegram group 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by nspinternet** [comment]
![111](https://github.com/hotspotbilling/phpnuxbill/assets/122086121/fe5c8888-67f3-48af-a2c1-1e36bf337661)

**Comment by nspinternet** [comment]
if i edit pppoe plan or i m adding new pppoe plan. then i m facing this error

**Comment by ibnux** [comment]
Update database

**Reply by nspinternet** [reply, marked as answer]
Sir I have also updated the database and checked but still same issue

**Reply by nspinternet** [reply]
![Screenshot_2024_0319_142551](https://github.com/hotspotbilling/phpnuxbill/assets/122086121/511bcf62-2565-4952-bb20-019ed86effd8)
